addons/calendar/models/calendar_event.py

```diff
@@ -81,14 +82,20 @@ def _default_partners(self):
                 partners |= self.env['res.partner'].browse(active_id)
         return partners
 
+    def _default_access_token(self):
+        return uuid.uuid4().hex
+
     # description
     name = fields.Char('Meeting Subject', required=True)
     description = fields.Html('Description')
     user_id = fields.Many2one('res.users', 'Organizer', default=lambda self: self.env.user)
     partner_id = fields.Many2one(
         'res.partner', string='Scheduled by', related='user_id.partner_id', readonly=True)
     location = fields.Char('Location', tracking=True, help="Location of Event")
-    videocall_location = fields.Char('Meeting URL')
+    videocall_location = fields.Char('Meeting URL', compute='_compute_videocall_location', store=True)
+    access_token = fields.Char('Invitation Token', default=_default_access_token, copy=False, required=True)
+    videocall_source = fields.Selection([('discuss', 'Discuss'), ('custom', 'Custom')], default="custom", required=True)
+    videocall_channel = fields.Many2one('mail.channel', 'Discuss Channel')
```

Comment:
Follow the convention.
```suggestion
    videocall_channel_id = fields.Many2one('mail.channel', 'Discuss Channel')
```